jdk/src/java.base/share/classes/jdk/internal/module/jdk8_packages.dat

```diff
@@ -1,3 +1,9 @@
+/*
+ * ===========================================================================
+ * (c) Copyright IBM Corp. 2015, 2017 All Rights Reserved
```

Comment:
Does this work? I think the IBM copyright should use the same comment character as the Oracle copyright. Also the dates should be 2018, 2018.